src/lib/core/typography/_typography.scss

```diff
@@ -42,17 +42,21 @@
 }
 
 // Adds the base typography styles, based on a config.
-@mixin mat-typography($config: mat-typography-config(), $selector: '.mat-typography') {
+// TODO(crisbeto): the headers need to handle margins as well.
+@mixin mat-base-typography($config, $selector: '.mat-typography') {
   .mat-h0, .mat-hero-header {
     @include mat-typography-level-to-styles($config, display-4);
+    letter-spacing: -0.05em;
```

Comment:
[I already left a link to the spec right above the pre-defined values.](https://github.com/angular/material2/pull/4375/files#diff-6eb6698b7e87b7f029b5578b595697d3L13)